services/frontend/src/store/sagas/profiles.sagas.js

```diff
@@ -13,8 +13,34 @@ function* profilesFetchAll() {
     yield put(profileSkillsSlice.actions.addProfileSkills(profileSkills));
 }
 
+function* profilesPatchPersonalDetails({payload}, success) {
+    try {
+        const result = yield call(api.service("profiles").patch, payload.id, payload);
+        const normalizedProfile = Profile.normalizeProfile(result);
+
+        yield put(profilesSlice.actions.setProfile(normalizedProfile));
+    }
+    catch (error) {
+        yield put(notificationRequestsSlice.setNotification.actions.request(
+            {type: "error", message: error.message}
+        ));
+        throw error;
+    }
+
+    yield put(notificationRequestsSlice.setNotification.actions.request(
+        {type: "success", message: "Updated successfully"}
+    ));
+    yield call(success);  // Mark success before continuing with other actions
```

Comment:
What's the reason to explicitly 'mark success before continuing with other actions'... when there are no other actions to continue with?

Reply:
a leftover comment from some copy paste shenanigans.

Deleted.